task-definition.json

```diff
@@ -0,0 +1,33 @@
+{
+  "family": "devnet-demo-client",
```

Comment:
Why is only this task definition in a file?

Reply:
Ah, maybe I have to explain, what that is. This is the AWS ECS task definition, which defines, how the docker image is deployed on AWS